### ISSUE TITLE
Unable to initiate Rollback on 5.4.145.9494

### ISSUE THREAD

**Issue by andersosthus** [Azure/service-fabric-issues#103]
Hi,

I have a test cluster that has been upgraded to 5.4.145.9494. I am now unable to initiate an Application Rollback on an upgrade in process.

```
Start-ServiceFabricApplicationRollback -ApplicationName 'fabric:/MyApp'
Start-ServiceFabricApplicationRollback : Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
```

I am able to start application upgrades just fine.

**Comment by masnider** [comment]
What version is the client at?

**Comment by andersosthus** [comment]
I tried from the Desktop Service Fabric Explorer and Powershell. Both should be the latest version that was released with the 5.3, but I can give exact versions tomorrow.

I was able to do a rollback earlier this week, and I have not changed the local versions of any of the SF parts since then.

Also, all other commands seems to be working fine. I can upload packages, register packages and start upgrades, just not start rollbacks.

**Comment by andersosthus** [comment]
The versions I'm using at my client to try and invoke a rollback are:
ServiceFabricExplorer (Desktop version): 5.3.311.9590

Service Fabric Powershell:
```
Get-Module | where {$_.Name -eq "ServiceFabric"} | select Name,Version
Name          Version
----          -------
ServiceFabric 3.1.0.0
```

**Comment by masnider** [comment]
Can you upgrade the client to the corresponding 5.4 SDK and see if the issue is resolved? I'll check also to see if there's some known issue.

**Comment by andersosthus** [comment]
Hi,

I've updated to the 5.4 SDK on my dev machine. I use the desktop version of the Service Fabric Explorer (5.4.145.9494).

I am still unable to initiate a rollback when an upgrade is in progress. The cluster is running v5.4.145.9494.

The error message is the same as above.

**Comment by mikanyg** [comment]
I have the same issue, unable to initiate a rollback. Cluster is latest version (5.4). Local SDK is the 5.3.*.301 release

**Comment by alexwun** [comment]
This looks like a bug in the ACL entries for the rollback request - we'll fix it in an upcoming update. In the meantime, it's also possible to rollback an upgrade by issuing a new upgrade command to the previous version, interrupting the current upgrade.

For example, if the application is upgrading from "v1" to "v2". Then the command:

Start-ServiceFabricApplicationUpgrade -ApplicationTypeVersion v1 <other arguments ... >

will trigger a rollback back to "v1", interrupting the current "v1" to "v2" upgrade. The major difference between this command and Start-ServiceFabricApplicationRollback is that the interrupting upgrade command will not return until the rollback has completed, where as the rollback command will return once the rollback has started.

**Comment by masnider** [comment]
Thanks folks, this was fixed and shipped as a part of to SF runtime 5.4CU2 (5.4.164).
